ggml.c

```diff
@@ -1783,6 +1912,16 @@ inline static void ggml_vec_gelu_f16(const int n, ggml_fp16_t * y, const ggml_fp
     }
 }
 
+inline static void ggml_vec_gelu_bf16(const int n, ggml_bf16_t * y, const ggml_bf16_t * x) {
+    for (int i = 0; i < n; ++i) {
+        union {
+            ggml_bf16_t f;
+            uint16_t i;
+        } u = {x[i]};
+        y[i] = ggml_table_gelu_bf16[u.i];
```

Comment:
Is there an advantage of using `union`, compared to what we do above in the `f16` version?

Reply:
The type aliasing in `ggml_vec_gelu_f16` is fine, I just like to type pun floats. In any case, this suggestion no longer applies, since your earlier suggestion allowed me to remove this code.